backend/vectorstore/supabase.py

```diff
@@ -25,6 +25,7 @@ def __init__(
     def similarity_search(
         self,
         query: str,
+        user_id: str = "none",
```

Comment:
why ?

Reply:
This was the bug we found no? is it not relevant any more?

Reply:
This was the big we previously had with the documents? I'll check it out today anyway.